envs/safety-gymnasium/safety_gymnasium/envs/tasks/push/push_level1.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""push level 1"""
```

Comment:
level 0 1 2 Specifies whether to describe the differences briefly.

Reply:
Yes, in corresponding class docstring. So they are omitted here.